platform/linux-generic/odp_traffic_mngr.c

```diff
@@ -1994,6 +1994,12 @@ static void tm_queue_cnts_decrement(tm_system_t *tm_system,
 	odp_atomic_sub_u64(&queue_cnts->byte_cnt, frame_len);
```

Comment:
Commit description: "determines" -> "considers"?

Reply:
Sure, "considers" it is.